src/styles/index.css

```diff
@@ -0,0 +1,123 @@
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: rgb(255, 255, 255);
+  max-width: 198px;
+  margin-bottom: 500px !important;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__image img {
+  width: 100%;
+}
+
+.card__title {
+  color: rgb(6, 11, 53);
```

Comment:
The color `rgb(6, 11, 53)` should be defined as an SCSS variable to maintain consistency across your stylesheets.